tests/test.ts

```diff
@@ -94,6 +94,9 @@ describe.skip('Test clasp pull function', () => {
 
 describe.skip('Test clasp push function', () => {
   it('should push local project correctly', () => {
+    fs.removeSync('.claspignore');
+    fs.writeFileSync('Code.js', ' ');
```

Comment:
When using `fs.writeFileSync` we need to make sure to at least write a blank space (so `' '` rather than `''` so that the files can be found here. https://github.com/google/clasp/blob/2cd12253a9c77dcd8c021a4949110dc1b9057b41/index.ts#L80

Reply:
Use a constant string instead of a mysterious `' '`. Like:
`const TEST_CODE_JS = 'function test() { Logger.log('test'); }`;